units/provisioning_create_host.go

```diff
@@ -280,10 +289,10 @@ retryLoop:
 				// the new image
 				if i == 0 {
 					buildingContainerJob := NewBuildingContainerImageJob(j.env, parent, imageURL, j.host.Provider)
-					grip.Debug(message.Fields{
-						"error":   j.env.RemoteQueue().Put(buildingContainerJob),
+					err = j.env.RemoteQueue().Put(buildingContainerJob)
+					grip.Error(message.WrapError(err, message.Fields{
```

Comment:
We typically don't log duplicate key errors at error level. Do you expect this not to happen/is this really an error?

Reply:
nope, debug was the right level to log at